formats/organisation.jsonnet

```diff
@@ -0,0 +1,571 @@
+              name: {
+                type: "string",
+              },
+              role: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              role_href: {
+                type: "string",
+                format: "uri",
+              },
+              image: {
+                "$ref": "#/definitions/image",
+              },
+              unpaid: {
+                type: "boolean",
+              },
+            },
+          },
+          description: "A list of the organisation's traffic commissioners, if appropriate. Turn into proper links once organisations are fully migrated.",
+        },
+        ordered_chief_professional_officers: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "name",
+              "role",
+              "href",
+            ],
+            properties: {
+              name_prefix: {
+                type: "string",
+              },
+              name: {
+                type: "string",
+              },
+              role: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              role_href: {
+                type: "string",
+                format: "uri",
+              },
+              image: {
+                "$ref": "#/definitions/image",
+              },
+              unpaid: {
+                type: "boolean",
+              },
+            },
+          },
+          description: "A list of the organisation's chief professional officers, if appropriate. Turn into proper links once organisations are fully migrated.",
+        },
+        ordered_special_representatives: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "name",
+              "role",
+              "href",
+            ],
+            properties: {
+              name_prefix: {
+                type: "string",
+              },
+              name: {
+                type: "string",
+              },
+              role: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              role_href: {
+                type: "string",
+                format: "uri",
+              },
+              image: {
+                "$ref": "#/definitions/image",
+              },
+              unpaid: {
+                type: "boolean",
+              },
+            },
+          },
+          description: "A list of the organisation's special representatives, if appropriate. Turn into proper links once organisations are fully migrated.",
+        },
+        ordered_contacts: {
```

Comment:
I think these also might be migrated, not sure though.

Reply:
Looks like they are published to the publishing-api, so moving them to links